### PR TITLE
Added image feature to Inline-Quotes

### LINKED ISSUE #4097: Add user images to the in-line testimonials
### Current Behavior
The Layer5 website has in-line testimonials on a few places, including https://layer5.io/cloud-native-management/meshmap/ and https://layer5.io/cloud-native-management/meshery.

image

### Desired Behavior
We need to add a picture of the user in these in-line quotes, similar to the Review slider at the end of the page https://layer5.io/cloud-native-management/meshmap. The image could be creatively placed beside the name of the user.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Saksham294**
**Description**

This PR fixes #4097 
This PR adds user images to inline testimonials

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Saksham294** [comment]
I have added the feature of user images to in-line testimonials. I couldn't find photos of two people so would need help in that regard, apart from that we can surely discuss about the design changes expected and I would do the needful.
Thanks

**Comment by l5io** [comment]
🚀 Preview for commit a6f4f840a0b1d73da523f08bcbde8c8b85881b18 at: https://643aa4cadbdcc42205059e09--layer5.netlify.app

File: src/components/Inline-quotes/index.js [hunk 1]
```diff
@@ -50,6 +51,12 @@ text-align: center;
     @media screen and (max-width: 500px) {
         text-align: center;
     }
+    img{
+        border-radius: 50%;
+        width: 90px;
+        height: 90px;
```
Comment (leecalcote) [inline src/components/Inline-quotes/index.js:57]: @Saksham294 let's make this responsive with use of max-height, %s, or some other dynamic CSS approach.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Thanks, @Saksham294! 👏

**Review by Shivam-AfA — COMMENTED** [review]
Thanks @Saksham294 for this! I wonder if it would look better if we place the image on the left hand side of the person's name, rather than on top of it.

**Comment by Saksham294** [comment]
Hey @leecalcote , I made the requested changes for the image tag and made it responsive.

**Comment by l5io** [comment]
🚀 Preview for commit 55d28507e6660af850a762ce9a2c2e796c32ced5 at: https://643d662b3c846f30a1a60e82--layer5.netlify.app

**Review by Shivam-AfA — COMMENTED** [review]
@Saksham294 It looks good to me. 

To aid the correct positioning of longer names and titles, you can try increasing padding around the name and title box.

**Comment by Saksham294** [comment]
@Shivam-AfA if you think the changes are fine, then I think we can merge this PR

**Comment by Saksham294** [comment]
Forgot to mention, we still need photographs of a few people in the testimonials

**Comment by Shivam-AfA** [comment]
Please correct the padding as well @Saksham294

**Comment by Saksham294** [comment]
Screenshot 2023-04-18 at 11 03 39 PM
Does this look fine?

**Comment by Saksham294** [comment]
Screenshot 2023-04-18 at 11 06 04 PM
Or this one?

**Comment by Saksham294** [comment]
@Shivam-AfA I have made the changes, kindly check. Thanks

**Comment by l5io** [comment]
🚀 Preview for commit 8cd5cf8c00b03e63e4a45ac34cea12752876bd52 at: https://643fa8d71f987c7c8ca0ea4f--layer5.netlify.app

**Comment by leecalcote** [comment]
@Shivam-AfA good to go?

File: src/components/Inline-quotes/index.js [hunk 2]
```diff
@@ -41,24 +42,36 @@ text-align: center;
     border-image-slice: 1 1 1 0;
     transition: 0.6s ease-in-out;
 }
+img{
+    border-radius: 50%;
+    width: 5rem;
+    height: 5rem;
+    @media screen and (max-width: 600px) {
```
Comment (Shivam-AfA) [inline src/components/Inline-quotes/index.js:49]: @Saksham294 Please remove this media query if it's not being used.

**Comment by Shivam-AfA** [comment]
@leecalcote Looks good to me. However, we don't have images of the users for all the quotes on the meshmap page. Is it fine? 

@Saksham294 While you're in there, could you add the user images to the quotes on the meshery page as well?  https://layer5.io/cloud-native-management/meshery

**Comment by Saksham294** [comment]
@Shivam-AfA Do we have images for these users? Also as mentioned earlier I can't find images of some users for the meshmap page too

**Comment by Shivam-AfA** [comment]
Yes we have the images for them @Saksham294

**Comment by Saksham294** [comment]
@Shivam-AfA I can't the remaining images for the meshmap page but meshery page is done

**Comment by Saksham294** [comment]
Some merge conflict happened, I don't know why

**Comment by l5io** [comment]
🚀 Preview for commit f93d00e57ba1997abedb2176c1d75f7ade582204 at: https://64428ed6092a1974410d30ca--layer5.netlify.app

**Comment by Shivam-AfA** [comment]
@Saksham294 Please correct the responsiveness as well.

image

Also using `width: 5rem` for the image isn't dynamic, please use something in terms of % or vw.

**Comment by leecalcote** [comment]
Still spilling over. 
Screenshot 2023-04-24 at 4 31 58 PM

**Comment by l5io** [comment]
🚀 Preview for commit a430484855f71641a36a5fa6837c36af89e255b0 at: https://6446ff69e5b1502751cba7b0--layer5.netlify.app

**Pull request merged**